src/main/groovy/org/shipkit/internal/util/PropertiesUtil.java

```diff
@@ -0,0 +1,24 @@
+package org.shipkit.internal.util;
+
+import org.shipkit.internal.notes.util.IOUtil;
+
+import java.io.File;
+import java.io.FileReader;
+import java.util.Properties;
+
+public class PropertiesUtil {
+
+    public static Properties readProperties(File properties) {
+        Properties p = new Properties();
+        FileReader reader = null;
+        try {
+            reader = new FileReader(properties);
+            p.load(reader);
+        } catch (Exception e) {
+            throw new RuntimeException("Problems reading properties file: " + properties);
```

Comment:
It would be good to rethrow the original exception. Properties object won't give us much if it fails due to eg. nonexistent file.